packages/api/src/__tests__/normalizeRequest.test.ts

```diff
@@ -62,7 +63,7 @@ test('Normalizes an aws event with base64', () => {
   )
 
   expect(normalizeRequest(corsEventB64)).toEqual({
-    headers: corsEventB64.headers,
+    headers: new Headers(corsEventB64.headers),
```

Comment:
Note @dthyresson - looks like we really ought to have done this before.

I noticed this when I moved it over, that we weren't transforming the Headers even though the type expected it to.